gulpfile.js

```diff
@@ -150,5 +160,5 @@ function log(error) {
 }
 
 
-gulp.task('default', ['dist', 'copy']);
+gulp.task('default', ['dist', 'copy', 'deploy']);
```

Comment:
Doesn't the `['copy', 'dist']` requirement on `deploy` guarantee that both of those two tasks have completed before `deploy` will run?  It shouldn't matter what order they're in.